docs/wiki/User Guide/REST-APIs.md

```diff
@@ -584,6 +589,7 @@ Supported parameters are:
 | disable_self_healing_for                      | list      | list of anomaly types to disable self-healing                             | N/A                   | yes       |
 | enable_self_healing_for                       | list      | list of anomaly types to enable self-healing                              | N/A                   | yes       |
 | concurrent_partition_movements_per_broker     | integer   | upper bound of ongoing replica movements into/out of a broker             | N/A                   | yes       |
+| max_partition_movements_in_cluster            | integer   | upper bound of maximum replica movements going into/out of brokers across cluster   | `default_config`  | yes |                                                            
```

Comment:
(Applies to other similar updates in tables) Can we ensure that the table is formatted properly by adding spaces to ensure that `|` characters are aligned vertically?